### ISSUE TITLE
unable to intercept"Break Delay"

### ISSUE THREAD

**Issue by CatTeaA** [Updated-NoCheatPlus/NoCheatPlus#300]
unable to intercept"Break Delay"
Mod：meteor-client   
Turn on <Break Dekay> to dig quickly.and cannot be intercepted
Request an increase in detection interception item
Build#164

**Comment by snoopchallenge** [comment]
to be fair i think if you had a standalone mod that removed the break delay i wouldnt count it as cheats

**Comment by Huge-mistake** [comment]
> to be fair i think if you had a standalone mod that removed the break delay i wouldnt count it as cheats

Ignoring the Break Delay is also obviously cheating, and can speed up the speed at which blocks are broken

**Comment by ManInMyVan** [comment]
I don't think this is actually detectable, since [this is possible on an unmodified client](https://github.com/GrimAnticheat/Grim/issues/1632)

**Comment by CatTeaA** [comment]
> I don't think this is actually detectable, since [this is possible on an unmodified client](https://github.com/GrimAnticheat/Grim/issues/1632)

I am currently using 'grimac' to intercept the 'Break Delay' feature of 'meteor-client', and it feels effective.
It can prevent players from mining blocks more quickly.
